SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java

```diff
@@ -942,6 +948,55 @@ protected boolean downloadAssetSynchronously(final String assetPath) {
         return false;
     }
 
+    protected boolean downloadFontAssetSynchronously() {
+
+        String cdnRoot = "http://content-cdn.swrve.com/messaging/fonts/";
```

Comment:
This is the default cdnroot which was already there, so I kept it in there. But I agree, and would prefer to remove  the default. Will do!!